### ISSUE TITLE
Time-Clock MOD mit Time Clock Pro

### ISSUE THREAD

**Issue by ECKCD** [svenbolte/time-clock#1]
Vielen Dank für die toll Arbeit. Gibt es eine Möglichkeit Time-Clock MOD mit Time Clock Pro zu verknüpfen?

Beste Grüße

**Comment by svenbolte** [comment]
Nein, leider nicht - der Mod hat eigene Weiterentwicklungen und einige Dinge entfernt, die nicht in das Originalplugin zurückgeflossen sind. Zusätzlich ist die "Raumbuchungs" Funktion mit Shortcode nnur in diesem Fork enthalten.
